Neos.Flow/Classes/Mvc/Exception/StopActionException.php

```diff
@@ -23,4 +25,5 @@
  */
 class StopActionException extends \Neos\Flow\Mvc\Exception
 {
+    public ActionResponse $response;
 }
```

Comment:
as the `StopActionException` is only created at one place, and immediately enriched with its action request, i would strongly prefer to make it mandatory in the constructor and also mutable. Also we should document _why_ the request is attached at all and what its meaning is.

The same applies for the `ForwardException`, which is also only created once.

_But_ if you feel this is to breaky right now and will mudden your change im find with creating an additional pr to fix this before the 9.0 release.

Additonally while were at it, to make our code more stable, it think its fair to say that these both kind of exceptions are final, as our framework would probably just break if one tries to inherit from them. I also dont think it makes sense for the  `StopActionException` to be non final just that the `ForwardException` can inherit from that. _Can_ be separate final objects imo.

Reply:
Good plan 👍  I would just do it I think, it's a major version and I doubt too many instantiate those.